.github/workflows/comment-on-pr.yml

```diff
@@ -0,0 +1,58 @@
+name: Comment on PR
+
+on:
+  workflow_call:
+    secrets:
+      gh_token:
+        required: true
+
+jobs:
+    comment-on-pr:
+        runs-on: ubuntu-latest
+        steps:
+          - name: Get Artifact URL & PR Info
+            env:
+              GITHUB_TOKEN: ${{ secrets.gh_token }}
+              OWNER: ${{ github.repository_owner }}
+              REPO: ${{ github.event.repository.name }}
+              WORKFLOW_RUN_EVENT_OBJ: ${{ toJSON(github.event.workflow_run) }}
+            run: |
+              PREVIOUS_JOB_ID=$(jq -r '.id' <<< "$WORKFLOW_RUN_EVENT_OBJ")
```

Comment:
as the name is previous, I always wonder what if this is the first iteration and there is no previous job ?